### PR TITLE
Added ternary operator in EuclideanGCD.java

### PR DESCRIPTION
**Pull request by Monal5031**
#### By submitting this pull request I confirm I've read and complied with the below requirements.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [ ] This pull request has a descriptive title. For example, `Added {Algorithm name} [{Language}]`, not `Update README.md` or `Added new code`.

There was a minor change requested for changing if-else to ternary operator.

### CODE REVIEW

File: euclidean_gcd/EuclideanGCD.java [hunk 1]
```diff
@@ -16,18 +16,16 @@ static int euclideanGCD(int first, int second) {
     }
 
     static int euclideanGCDRecursive(int first, int second) {
-        if(second == 0)
-            return first;       // First becomes gcd if second becomes zero
-        else
-            return euclideanGCDRecursive(second, (first % second));
+        return (second == 0) ? first : euclideanGCDRecursive(second, (first % second))
```
Comment (singhpratyush) [inline euclidean_gcd/EuclideanGCD.java]: You missed the semicolon. Please make sure that you run the code before commiting.

Comment (Monal5031) [inline euclidean_gcd/EuclideanGCD.java]: Oops! 😅

Comment (singhpratyush) [inline euclidean_gcd/EuclideanGCD.java]: Also, can you work up the issues mentioned in `CodeClimate` (except Javadoc)?

Comment (Monal5031) [inline euclidean_gcd/EuclideanGCD.java]: Okay, will do it.

File: euclidean_gcd/EuclideanGCD.java [hunk 2]
```diff
@@ -16,18 +16,18 @@ static int euclideanGCD(int first, int second) {
...
-        if(second == 0)
-            return first;       // First becomes gcd if second becomes zero
-        else
-            return euclideanGCDRecursive(second, (first % second));
+        return (second == 0) ? first : euclideanGCDRecursive(second, (first % second));
+                // First becomes GCD when second becomes zero
     }
 
-    public static void main(String args[]) {
+    public static void main(String []args) {
```
Comment (singhpratyush) [inline euclidean_gcd/EuclideanGCD.java]: Brackets to be attached with type descriptor.

**Comment by Monal5031** [comment]
@aviaryan Please review.

**Pull request merged**